db/models/comment.js

```diff
@@ -8,8 +8,8 @@ module.exports = (sequelize, DataTypes) => {
      * The `models/index` file will call this method automatically.
      */
     static associate(models) {
-      this.belongsTo(models.post);
-      this.belongsTo(models.user);
+      this.belongsTo(models.post, { as: "post", foreignKey: "postId" });
+      this.belongsTo(models.user, { as: "user", foreignKey: "userId" });
```

Comment:
Why changing the comment model? It's unnessary to use as and foreignKey if there are not alias.

```suggestion
      this.belongsTo(models.post);
      this.belongsTo(models.user);
```